src/config.js

```diff
@@ -105,7 +106,7 @@ export default Object.assign(
      * Module scheme to use. The module scheme will be used to combine module
      * set of the application.
      */
-    MODULE_SCHEME_NAME: 'vanilla',
+    MODULE_SCHEME_NAME: MODULE_SCHEME_NAME,
```

Comment:
should be reverted, the babel should read `MODULE_SCHEME_NAME` of the config.js. consider another workaround if you have any issues of getting the `MODULE_SCHEME_NAME` from here